src/test/java/com/example/demoSpringRestaurant/unit/service/RestaurantServiceTest.java

```diff
@@ -31,28 +35,92 @@ public class RestaurantServiceTest {
+
+        //assert / then
+        assertThat(restaurantDtoList).usingRecursiveComparison().isEqualTo(RestaurantFixture.getRestaurantDtoList());
+        verify(restaurantRepository, times(1)).findAll();
+        verifyNoMoreInteractions(restaurantRepository);
+    }
+
+    @Test
+    void updateRestaurantShouldUpdateOneRestaurant() throws EntityNotFoundException {
+        //arrange / given
+        when(restaurantMapper.fromEntityToRestaurantDto(any(RestaurantEntity.class)))
+                .thenReturn(RestaurantFixture.getRestaurantDto());
+        when(restaurantMapper.fromRestaurantUpdateDtoToEntity(any(RestaurantUpdateDto.class)))
+                .thenReturn(RestaurantFixture.getRestaurantEntity(false));
+        when(restaurantRepository.save(any(RestaurantEntity.class)))
+                .thenReturn(RestaurantFixture.getRestaurantEntity(true));
+        when(restaurantRepository.existsById(any(Long.class))).thenReturn(true);
+        //act / when
+        var restaurantUpdate = restaurantService.updateRestaurant(5L, RestaurantFixture.getRestaurantUpdateDto());
+        var restaurantUpdate2 = restaurantService.updateRestaurant(1L, RestaurantFixture.getRestaurantUpdateDto());
+
+        //assert / then
+        assertThat(restaurantUpdate.getId()).isEqualTo(1L);
+        assertThat(restaurantUpdate2.getId()).isEqualTo(1L);
+        assertThat(restaurantUpdate).usingRecursiveComparison().isEqualTo(RestaurantFixture.getRestaurantDto());
+        assertThat(restaurantUpdate).isEqualTo(restaurantUpdate2);
+        verify(restaurantRepository, times(2)).existsById(any(Long.class));
+        verify(restaurantRepository, times(2)).save(any(RestaurantEntity.class));
+        verifyNoMoreInteractions(restaurantRepository);
+    }
+
+    @Test
+    void updateParametersInRestaurantShouldUpdateOneRestaurant() throws EntityNotFoundException {
+        when(restaurantMapper.fromEntityToRestaurantDto(any(RestaurantEntity.class)))
+                .thenReturn(RestaurantFixture.getRestaurantDto());
+        when(restaurantMapper.fromRestaurantDtoToEntity(any(RestaurantDto.class)))
+                .thenReturn(RestaurantFixture.getRestaurantEntity(true));
+        when(restaurantRepository.save(any(RestaurantEntity.class)))
+                .thenReturn(RestaurantFixture.getRestaurantEntity(true));
+        when(restaurantRepository.findById(any(Long.class)))
+                .thenReturn(Optional.of(RestaurantFixture.getRestaurantEntity(true)));
+
+        var restaurantDto = restaurantService.updateParametersInRestaurant(5L, RestaurantFixture.getRestaurantUpdateDto());
+
+        assertThat(restaurantDto.getId()).isEqualTo(1L);
+        assertThat(restaurantDto).usingRecursiveComparison().isEqualTo(RestaurantFixture.getRestaurantDto());
+        verify(restaurantRepository, times(1)).findById(any(Long.class));
+        verify(restaurantRepository, times(1)).save(any(RestaurantEntity.class));
+        verifyNoMoreInteractions(restaurantRepository);
```

Comment:
Az ID itt is ugyanúgy fura, felesleges visszaellenőrizni, az a mockból került oda, nem a valódi implementáció működését ellenőrzöd vissza vele.

Illetve hibaeset tesztelés jó lenne itt is.